dev/VSIX/Extension/Cpp/Common/VSPackage.resx

```diff
@@ -151,58 +151,58 @@
   </data>
   <data name="1017" xml:space="preserve">
-    <value>Custom Control (WinUI 3)</value>
+    <value>Templated Control</value>
   </data>
   <data name="1018" xml:space="preserve">
-    <value>[Experimental] Custom Control (WinUI 3)</value>
+    <value>[Experimental] Templated Control</value>
   </data>
   <data name="1019" xml:space="preserve">
-    <value>A custom control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A blank custom control with the appropriate default styling (WinUI 3).</value>
   </data>
   <data name="1021" xml:space="preserve">
-    <value>User Control (WinUI 3)</value>
+    <value>User Control</value>
   </data>
   <data name="1022" xml:space="preserve">
-    <value>[Experimental] User Control (WinUI 3)</value>
+    <value>[Experimental] User Control</value>
   </data>
   <data name="1023" xml:space="preserve">
-    <value>A user control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A blank user control (WinUI 3).</value>
   </data>
   <data name="1025" xml:space="preserve">
-    <value>Blank App, Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)</value>
+    <value>WinUI Blank App (Packaged with Windows Application Packaging Project)</value>
   </data>
   <data name="1026" xml:space="preserve">
-    <value>[Experimental] Blank App, Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)</value>
+    <value>[Experimental] WinUI Blank App (Packaged with Windows Application Packaging Project)</value>
   </data>
   <data name="1027" xml:space="preserve">
-    <value>A project template for creating a Desktop app based on the Windows UI Library (WinUI 3). A Windows Application Packaging (WAP) project is included to create a MSIX package for side-loading or distribution via the Microsoft Store.</value>
+    <value>A project template for creating a WinUI 3 app. A Windows Application Packaging (WAP) project is included to create a MSIX package for side-loading or distribution via the Microsoft Store.</value>
   </data>
   <data name="1029" xml:space="preserve">
-    <value>Blank App, Packaged (WinUI 3 in Desktop)</value>
+    <value>WinUI Blank App (Packaged)</value>
   </data>
   <data name="1030" xml:space="preserve">
-    <value>[Experimental] Blank App, Packaged (WinUI 3 in Desktop)</value>
+    <value>[Experimental] WinUI Blank App (Packaged)</value>
   </data>
   <data name="1031" xml:space="preserve">
-    <value>A project template for creating a Desktop app based on the Windows UI Library (WinUI 3) along with a MSIX package for side-loading or distribution via the Microsoft Store.</value>
+    <value>A project template for creating a WinUI 3 app along with a MSIX package for side-loading or distribution via the Microsoft Store.</value>
   </data>
   <data name="1033" xml:space="preserve">
-    <value>Windows Runtime Component (WinUI 3)</value>
+    <value>Windows Runtime Component</value>
```

Comment:
Any specific reason for removing WinUI3 as I see there are windows runtime components for UWP as well and keeping WinUI 3 might help to distinguish.